src/sentry/static/sentry/app/__mocks__/api.jsx

```diff
@@ -41,6 +41,11 @@ class Client {
       return url === response.url && (options.method || 'GET') === response.method;
     });
   }
+
+  uniqueId() {
+    return '123';
```

Comment:
what even is the unique ID thing used for? I think @macqueen also implemented this in a recent PR so you may not need to do this in yours

Reply:
Not sure, looks like it's used in the action creators in API client